frontend/revive-gallery-app/src/App.js

```diff
@@ -10,7 +10,7 @@ function App() {
   return (
     <Router>
       <Routes>
-          <Route path="/" element={<LoginForm />} />
+          <Route path="/" element={<RegistrationForm />} />
```

Comment:
@anjaliputta It is better to create separate component for the home page

Reply:
@SaiKiran0407 Sure, I am working on that task. I will create a separate PR for that.